package.json

```diff
@@ -83,5 +84,11 @@
         "document-model": "^1.0.5",
         "react": "^18.2.0",
         "react-dom": "^18.2.0"
+    },
+    "dependencies": {
```

Comment:
This lib shouldn't have dependencies other than the peerDependencies listed above, as any dependencies would have to be installed in the Connect app. These dependencies should be moved to ´devDependencies´.

So every lib should bundle its code together with the dependencies. We configure the only external dependencies here: https://github.com/powerhouse-inc/document-model-libs/blob/744a37c4bea4d2d1fae6762fb0592d5a7068f99a/vite.config.ts#L23